test/clj/game_test/core.clj

```diff
@@ -111,6 +111,10 @@
   [state pos]
   (get-in @state [:runner :rig :facedown pos]))
 
+(defn get-discarded
+  [state side pos]
+  (get-in @state [side :discard pos]))
```

Comment:
I'd add a "default" value for pos as well for when you get the top discarded card.